.travis.yml

```diff
@@ -10,20 +10,19 @@ addons:
   chrome: stable
 
 cache:
-  yarn: true
+  directories:
+    - $HOME/.npm
 
 env:
   global:
     # See https://git.io/vdao3 for details.
     - JOBS=1
 
 before_install:
-  - curl -o- -L https://yarnpkg.com/install.sh | bash
-  - export PATH=$HOME/.yarn/bin:$PATH
-
-install:
-  - yarn install --no-lockfile --non-interactive
+  - npm install -g npm@6
+  - npm config set spin false
 
 script:
-  - yarn lint:js
-  - yarn test
```

Comment:
I think I'm missing some context here, did we end up deciding we were going to shift **away** from Yarn? I'd thought our decision had been to stick **with** yarn as a team last I knew ...

Reply:
In general, we try to use yarn, however it appears that this project already uses npm (based on the lockfiles) so this doesn't appear to be a "change" to me. I added an approving review to move things along, but it would be good to have the rationale recorded here